internal/transport/http.go

```diff
@@ -1,15 +1,15 @@
-package ui
+package transport
 
 import (
 	"errors"
 	"log"
 	"net/http"
 
 	"github.com/evt/port-api/internal/app"
-	"github.com/evt/port-api/internal/app/adapters"
 	"github.com/evt/port-api/internal/app/command"
 	"github.com/evt/port-api/internal/app/query"
 	"github.com/evt/port-api/internal/common/server"
+	"github.com/evt/port-api/internal/repository"
```

Comment:
Similarly to my previous comment you are tightly coupling package `transport` to `repository`, this bypass completely the `core`, which nullify a large benefit of hexagonal architecture

Reply:
fixed

Reply:
fixed by moving sentinel errors to domain package, cheers